### PR TITLE
add login, register, user management

### PR DESCRIPTION
**Pull request by yaohua-H**

### CODE REVIEW

File: cmd/admin/controller/account/guest.go [hunk 1]
```diff
@@ -0,0 +1,121 @@
+package account
+
+import (
+	"github.com/dgrijalva/jwt-go"
```
Comment (AlexStocks) [inline cmd/admin/controller/account/guest.go:27]: format ur imports by https://github.com/dubbogo/tools/blob/master/cmd/imports-formatter/main.go

Comment (yaohua-H) [reply cmd/admin/controller/account/guest.go:27]: done

**Comment by ztelur** [comment]
pl make ci successful

File: cmd/admin/controller/utils.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ */
+
+package controller
+
+import (
+	"github.com/dubbogo/pixiu-admin/pkg/config"
+)
+
+// Version admin version
+const Version = "0.1.0"
```
Comment (AlexStocks) [inline cmd/admin/controller/utils.go]: 用如下形式把所有常量都包含在内：
const (

)

File: cmd/admin/controller/utils.go [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+const OK = "10001"
+const ERR = "10002"
+const RETRY = "10003"
+
+const ResourceId = "resourceId"
```
Comment (AlexStocks) [inline cmd/admin/controller/utils.go]: 不要用 Id，专有名词如 ID/TCP/UDP/HTTP/FTP都统统用大写

File: cmd/admin/controller/utils.go (hunk 2 above)
Comment (yaohua-H) [reply cmd/admin/controller/utils.go]: done

File: cmd/admin/controller/utils.go (hunk 3 above)
Comment (yaohua-H) [reply cmd/admin/controller/utils.go]: done

**Review by AlexStocks — APPROVED** [review]

File: cmd/admin/controller/account/guest.go [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+	"github.com/dubbogo/pixiu-admin/cmd/admin/controller/auth"
+	"github.com/dubbogo/pixiu-admin/pkg/logic/account"
+	"github.com/dubbogo/pixiu-admin/pkg/utils"
+)
+
+func Register(c *gin.Context) {
+	username := c.PostForm("username")
+	password := c.PostForm("password")
+
+	// username password Format check
```
Comment (mark4z) [inline cmd/admin/controller/account/guest.go:45]: Is this commented out code necessary?

File: cmd/admin/controller/account/user.go [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+	c.JSON(http.StatusOK, controller.WithRet(token))
+}
+
+// EditPassword modify account password
+func EditPassword(c *gin.Context) {
+
+	oldPassword := c.PostForm("oldPassword")
+	newPassword := c.PostForm("newPassword")
+
+	//if flag, _ := regexp.MatchString("^(?![0-9]+$)(?![a-zA-Z]+$)[0-9A-Za-z]{6,20}$", oldPassword); !flag {
```
Comment (mark4z) [inline cmd/admin/controller/account/user.go:63]: same problem

**Review by mark4z — COMMENTED** [review]
Full Stack Developer？ nice job,

File: cmd/admin/controller/account/guest.go (hunk 4 above)
Comment (yaohua-H) [reply cmd/admin/controller/account/guest.go:45]: The web-side project was implemented by another developer, so the implementation of this function has not been discussed yet

**Review by mark4z — APPROVED** [review]
LGTM

**Review by ztelur — APPROVED** [review]

**Pull request merged**